src/Containers/ImageCarousel.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Times } from 'styled-icons/fa-solid/Times';
+import { flex, position, transition, scroll } from '@Utils/Mixins';
+import styled from 'styled-components';
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import { ImplicitPropsInterface } from '@Utils/Hooks';
+
+export interface ImageCarouselProps
+    extends MainInterface,
+        ResponsiveInterface,
+        ImplicitPropsInterface {
+    imageData: string[];
+    onClick: Function;
+    altText: string;
+    width?: number;
+    height?: number;
+}
+
+export const ImageCarousel: React.FC<ImageCarouselProps> = ({
+    imageData,
+    onClick,
+    altText,
+    width = 150,
+    height = 75,
+    ...props
+}) => {
+    return (
+        <Items {...props}>
+            {imageData.map(image => (
+                <Item key={image} onClick={() => onClick(image)}>
+                    <Overlay>
+                        <Icon /> Delete
+                    </Overlay>
+                    <img
+                        width={width}
+                        height={height}
+                        alt={altText}
```

Comment:
This will have the same alt for every image is that right?

Reply:
Yeah I was thinking it would just say either Store Image, or Store Logo